conan/cli/commands/version.py

```diff
@@ -12,10 +12,21 @@ def version(conan_api, parser, *args):
     Give information about the Conan client version.
     """
 
-    return {'version': str(conan_version),
+    return {
+            'version': str(conan_version),
+            'conan_script_path': sys.argv[0],
```

Comment:
```suggestion
            'conan_path': sys.argv[0],
```
Conan might not always be a script, isn't it? This might return the name of the ``conan.exe`` sometimes?